### PR TITLE
TE-2.1: gRIBI IPv4 Entry : Adding ipv6 entry for this testcase

### PR DESCRIPTION
**Pull request by avaneesh90**
TE-2.1: gRIBI IPv4 Entry : Adding ipv6 attribute entry for this testcase

### CODE REVIEW

**Comment by xw-g** [comment]
Can you share some insight of why to add the IPv6 addresses? I see it's not part of the README? Thanks.

File: feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go [hunk 1]
```diff
@@ -57,36 +57,48 @@ var (
...
 	}
 	atePort3 = attrs.Attributes{
 		Name:    "atePort3",
 		Desc:    "ATE Port 3",
 		IPv4:    "192.0.2.10",
+		IPv6:    "2001:db8::192:0:2:a",
 		IPv4Len: 30,
+		IPv6Len: 126,
 	}
 )
```
Comment (sachendras) [inline feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go:103]: What happens if we just remove the IPv6 addresses here and instead just enable IPv6 under the interface? Will that pass the test?

Comment (avaneesh90) [reply feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go]: I have removed the configuring ipv6 address and added enabling ipv6 in the interface

**Review by sachendras — APPROVED** [review]

**Comment by sachendras** [comment]
Internal Google agreement in b/264809673

**Pull request merged**